src/main/java/com/eleks/academy/whoami/configuration/ContextConfig.java

```diff
@@ -17,4 +21,14 @@ Server server(ServerProperties serverProperties) throws IOException {
         return new ServerImpl(serverProperties.port(), serverProperties.players());
     }
 
+    @Bean
```

Comment:
please try only using annotation `@CrossOrigin` from my last experience if I remember it correctly it should be enough, no need of extra configuration

Reply:
It really works without all of that's)